src/ontology/uberon-edit.obo

```diff
@@ -224008,6 +224008,53 @@ relationship: part_of UBERON:0000101 ! lobe of lung
 relationship: proximally_connected_to UBERON:0002184 ! segmental bronchus
 property_value: dcterms-date "2023-03-14T14:54:15Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600010
+name: bronchial submucosal gland ciliated duct
+def: "A ciliated duct in a bronchial submucosal gland." [PMID:30864819]
+intersection_of: UBERON:8600014 ! submucosal gland ciliated duct
+intersection_of: part_of UBERON:0002185 ! bronchus
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+relationship: dc-contributor https://orcid.org/0000-0002-9185-3994
+property_value: dcterms-date "2023-05-02T08:07:56Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600011
+name: tracheal submucosal gland ciliated duct
+def: "A ciliated duct in a tracheal submucosal gland." [PMID:30864819]
+intersection_of: UBERON:8600014 ! submucosal gland ciliated duct
+intersection_of: part_of UBERON:0003126 ! trachea
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+relationship: dc-contributor https://orcid.org/0000-0002-9185-3994
+property_value: dcterms-date "2023-05-02T08:08:14Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600012
+name: submucosal gland acinus
+def: "A spherical or tubular structure in a submucosal gland that produces and secretes mucus and other substances into the ducts leading to the lumen." [PMID:17707699, PMID:25648514, PMID:30864819]
+is_a: UBERON:0011148 ! submucosal gland
+intersection_of: UBERON:0009842 ! glandular acinus
+intersection_of: part_of UBERON:0011148 ! submucosal gland
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date "2023-05-02T08:57:27Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600013
+name: submucosal gland collecting duct
+def: "A non-secretory tubular structure in the submucosal gland that collects and transports secretions towards the lumen." [PMID:17707699, PMID:25648514, PMID:30864819]
+is_a: UBERON:0011148 ! submucosal gland
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+property_value: dcterms-date "2023-05-02T08:08:59Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600014
+name: submucosal gland ciliated duct
+def: "A ciliated tubular structure connecting the collecting duct with the lumen." [PMID:17707699, PMID:30864819]
```

Comment:
Text definition appears to be incomplete.
- No mention of 'submucosal gland'
- connects 'collecting duct' (of submucosal gland? of kidney?) to lumen (lumen of what structure?)

Reply:
I agree with the change of lumen for surface epithelium, thank you.

> Of note, the dbxrefs are also specific to airway submucosal glands.

Yes, I will add more general dbrrefs.

Reply:
I have found [PMID:10520827 for collecting duct in esophagus SMGs](https://doi.org/10.1016/S0002-9270(99)00478-5) behind a paywall, but I can quote:

> "Interestingly, the ostia of the collecting ducts of the SMGs in the human esophagus can be identified on the luminal surface by staining with the dye toluidine blue. When this is done, the ostia at least appear to be organized into longitudinal rows parallel to the long axis ofthe organ, and this same pattern is apparent when the collecting ducts fill with barium on an upper gastrointestinal series in patients with the condition known as esophageal intramural pseudodiverticulosis."